### DISCUSSION TITLE
About requests to the rest api

### DISCUSSION THREAD

**Discussion by cengizili** [subzeroid/aiograpi-rest#60] [Announcements]
First of all thanks for creating such a solid framework. I am kind of new to these concepts so sorry if that's a silly question. Say I deployed this repo to a google cloud service and base url for the api is now the service's url. Further, there are requests made to that url from multiple sources with different ip addresses. In that case, source of the traffic to instagram is the ip address of the virtual machine that deployed service runs on or ip addresses where requests come from?

**Comment by adw0rd** [comment]
ip address of the virtual machine

**Reply by cengizili** [reply]
Is there a way to direct requests to instagram without interrupting? Using requesters' ip addresses is more practical than using proxies for my case.
